story: add release notes and documentation extended dashboard functionality 12.1 (#710)
valtimo-platform/valtimo-documentation: reference/modules/dashboard.md

#### Query condition

| Name            | Description                                                                                                                                                                                                                                                           |
|-----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `queryPath`     | The path of the variable that the condition uses to filter the count.                                                                                                                                                                                                 |
| `queryOperator` | The operator that the condition uses to filter the count. Available values are `!=`, `==`, `>`, `>=`, `<` and `<=`.                                                                                                                                                   |

> | `queryValue`    | The value which the queryPath variable is checked against. As of version 12.1, date time expressions can be used. For example: `${localDateTimeNow.minusWeeks(2)}`. This is useful for, for example, retrieving the amount of cases created in the past two weeks.    |

Reviewer: ```suggestion
| `queryValue`    | The value which the queryPath variable is checked against. As of version 12.1, date time expressions can be used. For example: `${localDateTimeNow.minusWeeks(2)}`. This is used to retrieve the amount of cases created in the past two weeks.    |
```

Front-end module: `CaseGroupByDataSourceModule`

| Name                 | Description                                                                                                                                                                                               |
|----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `documentDefinition` | Required. The key of the document definition for which cases need to be counted.                                                                                                                          
| `path`               | Required. The path of which the values need to be grouped. For example `case:createdBy`.                                                                                                                  
| `queryConditions`    | Optional. An array of conditions that the case needs to match in order to be included in the counts. Each item in the array is a query condition object described in the table under `case-count`.        |

> | `enum`               | Optional. An object which defines how items retrieved by the group by need to be displayed. This can be used to show technical values in a user-friendly way. For example `{"isRequired": "Is required"}` |

Reviewer: ```suggestion
| `enum`               | Optional. An object which defines how items retrieved by the group by need to be displayed. This can be used to show technical values in a user-friendly way. For example `{"isRequired": "Is required"}`. |
```

| `highSeverityThreshold`   | Limit above which the color of the number becomes red.                                                                                                                                                                             |

### Donut chart

Key: `donut`

Front-end module: `DonutModule`

Displays multiple counts in a donut chart.

![Example of 'donut' display type of widget](img/display-type-donut-example.png)

#### Properties

| Name                      | Description                                                                                               |
|---------------------------|:----------------------------------------------------------------------------------------------------------|
| `title`                   | Required. The title of the widget, shown at the top of the widget.                                        |
| `subtitle`                | The text shown below the title.                                                                           |

> | `label`                   | The label that is displayed in the center of the donut. | |

Reviewer: ```suggestion
| `label`                   | The label that is displayed in the center of the donut.  |
```

| `subtitle`                | The text shown below the title.                                                                                                                                                                                                    |

### Gauge

Key: `gauge`

Front-end module: `GaugeModule`

Displays a count and a total count in a gauge.

![Example of 'gauge' display type of widget](img/display-type-gauge-example.png)

#### Properties

| Name                      | Description                                                                              |
|---------------------------|:-----------------------------------------------------------------------------------------|
| `title`                   | Required. The title of the widget, shown at the top of the widget.                       |
| `subtitle`                | The text shown below the title.                                                          |

> | `label`                   | The label specifies the type of item under the gauge. In the screenshot this is 'zaken'. | |

Reviewer: ```suggestion
| `label`                   | The label specifies the type of item under the gauge. In the screenshot this is 'zaken'. |
```
